auto_landing/launch/auto.launch

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<launch>
+    
+    <arg name="mav_name" default="firefly" />
+	<arg name="x" value="2.0"/>
+    <arg name="y" value="3.0"/>
+	
+	<include file="$(find rotors_gazebo)/launch/mav.launch" />
+
+    <include file="$(find husky_gazebo)/launch/spawn_husky.launch" >
+	    <arg name="x" value="$(arg x)"/>
+        <arg name="y" value="$(arg y)"/>
+        <arg name="z" value="0.3"/>
+		
```

Comment:
fix indentation. try to keep all value tags in the same vertical line, this increase readability a lot, you can look at the planning_library launch files.

Reply:
done